templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul class="sidebar-nav">
+  <li><a class="links" href="{% url 'taxi:index' %}">Home page</a></li>
+  <li><a class="links" href="#">Manufacturers</a></li>
+  <li><a class="links" href="#">Cars</a></li>
+  <li><a class="links" href="#">Drivers</a></li>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are placeholders. Please update them with actual URLs using the `{% url %}` template tag once the corresponding views and URL patterns are defined.